cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,74 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
-describe('Settings page', () => {
-  before(() => {
+import SettingsPageObject from "../support/pages/settings.pageObject";
+import HomePageObject from "../support/pages/home.pageObject";
+import ProfilePageObject from "../support/pages/profile.pageObject";
+import SignInPageObject from "../support/pages/signIn.pageObject";
 
-  });
+const settingsPage = new SettingsPageObject();
+const homePage = new HomePageObject();
+const profilePage = new ProfilePageObject();
+const signInPage = new SignInPageObject();
 
-  beforeEach(() => {
+describe('Settings page', () => {
+  let user;
+  const bio = 'New bio information';
+  const updatedEmail = 'inna@qa.team';
+  const updatedPassword = 'Inna1234!';
+  const email = 'riot@qa.team';
+  const username = 'riot';
 
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
   });
 
-  it('should provide an ability to update username', () => {
-
+  it('should update username', () => {
+    cy.login();
+    settingsPage.visit();
```

Comment:
```suggestion
    cy.login();
    settingsPage.visit();
```
you could move it to the beforeEach method